sklearn/metrics/tests/test_score_objects.py

```diff
@@ -85,11 +85,15 @@ def _require_positive_y(y):
     return y
 
 
-def _make_estimators(X_train, y_train, y_ml_train):
+def _make_estimators(X_train, y_train, y_ml_train, y_reg_train=None):
     # Make estimators that make sense to test various scoring methods
     sensible_regr = DecisionTreeRegressor(random_state=0)
     # some of the regressions scorers require strictly positive input.
-    sensible_regr.fit(X_train, y_train + 1)
+    if y_reg_train is None:
+        sensible_regr.fit(X_train, y_train + 1)
```

Comment:
`y_train + 1` is to get only positive value? It seems that `_require_positive_y` would be much better no?

Reply:
I think so, not sure exactly what the purpose of sometimes using values >=0 and other times using values >=1. Will amend.

Reply:
@glemaitre Amended but maybe ` + 1` is faster?